text/1008-tls-and-alpn.rst

```diff
@@ -149,107 +150,105 @@ Client-side Server Certificate Verification
 ===========================================
 
 On the client side (both the language bindings and the REPL), TLS server
-certificate verification should always be enabled. In order to accept
-the self-signed certificate, at the time of certificate generation, the
-EdgeDB CLI will also copy the generated certificate into the so-called
-``credentials.json`` - a group of JSON files named after the EdgeDB
-instance in a well-known place (e.g. ``~/.edgedb/credentials/``
-depending on the OS) that are meant to store credentials for the client
-to establish connections to the EdgeDB instance. For example::
+certificate verification should always be enabled. By default, the
+system-wide trusted CA certificates are usually used to verify server
+certificates. For server certificates that are signed by untrusted CA,
+the users could provide the path to the specific CA certificate file
+they trust - for CLI the option is ``--tls-ca-file``, for language
+bindings the option is usually ``tls_ca_file`` or ``tlsCaFile``.
+
+In order to accept the self-signed certificate, at the time of
+certificate generation, the EdgeDB CLI will also copy the generated
+certificate into the so-called ``credentials.json`` - a group of JSON
```

Comment:
We already have `/home/yury/.config/edgedb/credentials/foo.json` --- when you say `credentials.json` do you mean `credentials/<instance-name>.json`?

Reply:
Yes, I should probably remove the double quotes for inline code block style, and refer to it as just "credentials JSON files"